{"name": "Невозможность изменения данных неавторизованного пользователя, name", "status": "passed", "description": "При попытке изменения {field} возвращает HTTP 401 и сообщение об ошибке", "steps": [{"name": "Отправка POST на auth/login", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'troywarner@example.org', 'password': '6538702972', 'name': 'Sean Harrell III'}"}], "start": 1718207482939, "stop": 1718207483209}, {"name": "Отправка PATCH на auth/user", "status": "passed", "parameters": [{"name": "payload", "value": "{'email': 'troywarner@example.org', 'password': '6538702972', 'name': 'abracadabra'}"}, {"name": "access_token", "value": "''"}], "start": 1718207483209, "stop": 1718207483439}], "parameters": [{"name": "field", "value": "'name'"}], "start": 1718207482939, "stop": 1718207483439, "uuid": "0bdffad0-c141-4d5a-aeaa-a132d0bd684e", "historyId": "19a34d938f393360e3a887d22b96cda7", "testCaseId": "f450658db434b016c441fc12e73231b2", "fullName": "tests.test_auth_user.TestChangeUserData#test_change_user_data_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestChangeUserData"}, {"name": "host", "value": "XPMEPODW0067"}, {"name": "thread", "value": "9424-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}